packages/@okta/i18n/src/properties/login.properties

```diff
@@ -819,6 +819,10 @@ oie.verify.google_authenticator.otp.title = Verify with Google Authenticator
 oie.verify.google_authenticator.otp.description = Enter the temporary code generated in your Google Authenticator app
 oie.google_authenticator.otp.enterCodeText = Enter code
 
+## Number Challenge
+oie.numberchallenge.warning = Haven't received a push notification yet? Try opening the Okta Verify app on your device or <$1>resend the push notification</$1>.
+oie.numberchallenge.subtitle = On your mobile device, open the Okta Verify prompt, then tap <$1>{0}</$1> in Okta Verify to continue.
```

Comment:
Love seeing this tag syntax used!

nit: For positional arguments, let's add a comment above the key for our translators to know what it is in reference to.